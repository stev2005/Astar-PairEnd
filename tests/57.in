GTTCTCTGCAGCCCATGTGTTCGGTCACATATTTAGCTAGTTTAATTGGGCCTATGCTTGTTCGGCATACCTAATATGGCAATTAGCCGATCTATGCTCCATAATTATGTAGTTGTGTAAAACTAACGTTTAGAACTCACACATAGGGCGGCGTCTTGCTCATTATCAACCAGCTTTCCCAAGGTTCTGGGTTCTTAATCCCCAGCTGGATGACCACGTTCCGAGCTGCTTTACTGGGAAGTTTCATGGGCTTTTCCTTGGTCGAAAGGCGCACACC
15
TACATAATCCGATATCGTCGCATACACCCTACTCCGCCCGGTGGTGTCCAATATTCTAGCTGGGTTCACGGTGCCCTATACTTCGGTGT
TGACCTAGACAGCGTGAGTTTA
GCACAGAGCGGGTCTCCAGACATAAGTTCTCTAGACAATGTCAGGTAAAGACGTCTCAGGTAGAGGCGTAAGCCATAATAGTGCTTAAAGTTGCTATATGA
ACACCCCACAATCTTCTTTGGGTGGTGCCTCGACTCGAGAAGTCCGGACGTAAGGGCAATTCCAGACCATCCCAG
TCTATGATAGCGTCCGGGGCGAGGTTTCTGTGTGTGATCCGGTCAATGGGTAGCGCAGGAAGGTACCCCGGTACAACTTTCGTTACCATTATCTGGATTCCCAGGAGTACGCACCAGGAGCCCTAACATAGCCGTTTCGTTAATCCTTTTCCATAATCCGCTTGCGCAATTTTAGACCTCACGCAGCTTGCACCGTTTTTGTGCTACACACGCTTCCAAATGAATGTGCTGTGAACCC
GGTATGTCAACATTCTGTTCCGTAATCCAGGTACTAATCATGAGGCCACACGGAGGATTAGGTGTTGTGTTCGATACAAGCGATGTGGGCTATGTTCCGTCGATGTTTATGCCGAACGGAACATATGCCCCGTCCCCAACTGGCCGCGCTGCCTACGTGTATCTCGAGGATGTCTATAGATCCAAGCGCAGCTTAAGAGAACGTGCTGCGGCTACTGTCTTTAGTATCA
CATAACCTTTCGAAGCTAACTTATGTTCATGGTCGTTTTGGAAGCGAAGACCAGAGCTTGGCAGTGCGCAAACCGGTGGGTATTGACT
ACGCCAATCGCGTCTATCTGTCCTCACCCCACGGGAGTTACA
ACCACGATCCACGAGTCTAGGGAGCTCCCGTCTAGAGGTTTTAGTTCAGCGAATGCGAGTGCCCCTGTCCGACTGAGACACTCC
TGCTAACCCTGACCCGAGTTAACACCGAGTTGGACGATAG
GGTAACAGCTGCATGCCGAACGGCTCGGCAATGTTGCTAGGTTGGCATTATCGGGCTAAAAATTATCCGGTCCGTTTTGGTCTCTGTGGTGTTGGTCTATCTAGCACATTTTCGAAATTTGCGGTACATGACCATTACTAAGTCCTGCTCACGGTGGCGGTGTATGTTTTTACGCGGTACAAGTTGCCTTTGGGGCCGAACAAGGGCACCCGCACAGG
CCCTTACACTCCACGTTAAACCCTGCATTGCAAGTTGCATACCCGTACTACACTATAAGATTATGAGA
GACTCGCAGTTGATTTTTAACATCAGTAATCTTTGACAAATTGTTGTGCTGGTCTTTGAAGCTCACGGCGGAGAACGTTTGGGCTCATAATGGGTGTCACAGCGTCTCAGCGATTAAATAAGGGAGCTTCATAGCTTAAATCGTAGTCGGCGAACAGGACTACTGGGGTTGGAACCGCGATTTATAACGATCCACTGTCCGTTTACAGGGTCCGCCGGGAGCCTCGTAGACAACTAGTGACC
TTTACCGCTATTCCGCTGTAGATGGT
GAGTTGGTTTGCTGAGTCACCGACCAATGTAAATTTCGTACCCATCGGTGTACTCGTCCCCGCCCCAGTTGAAACTCTGAGCATCCCATGGAATCCACTTCGACGCCTATAGAACCGGGCCACCTGTATGTAATGGATGCGGTGGCAATGCAGGGCACCATCAAAGGCCAGTGCGCGGC
