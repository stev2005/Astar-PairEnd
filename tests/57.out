41
7
46
36
117
106
39
18
39
18
107
31
119
10
88
